cogstat/cogstat_hyp_test.py

```diff
@@ -431,14 +435,26 @@ def variable_pair_hyp_test(data, x, y, meas_lev, normality=None):
                            ': BF<sub>10</sub> = %0.*f, BF<sub>01</sub> = %0.*f\n' % \
                            (non_data_dim_precision, bf10, non_data_dim_precision, 1/bf10)
 
-
             r, p = stats.spearmanr(data[x], data[y])
             population_result += _("Spearman's rank-order correlation") + \
                                  ': <i>r<sub>s</sub></i>(%d) = %0.*f, %s' % \
                                  (df, non_data_dim_precision, r, print_p(p))
 
-        else:  # TODO or normality couldn't be calculated
-            population_result += '<decision>'+_('Interval variables.') + ' ' + _('Normality violated.') + ' >> ' + \
+        elif normality is None or homoscedasticity is None:
+            # TODO warning instead of omitting hypothesis tests?
```

Comment:
I agree that, at the moment, a warning would be better, and hypothesis tests could be calculated.

Reply:
Modified to warning instead of decision.